src/compiler/diagnosticMessages.json

```diff
@@ -1701,7 +1701,14 @@
         "category": "Error",
         "code": 2653
     },
-
+    "Proper external module that carries external typings cannot contain tripleslash references.": {

```

Comment:
External package cannot contain tripleslash references. Please contact the package author to update the package typings.